tests/ubiconfig/test_ubi.py

```diff
@@ -32,12 +32,25 @@ def ubi7_config_file():
     return ubi7
 
 
+@pytest.fixture
+def invalid_config_file():
+    invalid = open(os.path.join(TEST_DATA_DIR, 'bad_configs/invalid_config.yaml'))
```

Comment:
with open () as f:
   yield f 

seems to be better

Reply:
hm, yep.